charts/airbyte-server/templates/deployment.yaml

```diff
@@ -187,8 +187,44 @@ spec:
               name: {{ .Release.Name }}-airbyte-env
               key: JOBS_DATABASE_MINIMUM_FLYWAY_MIGRATION_VERSION
         {{- end }}
+        # Values from secret
+        {{- if .Values.secrets }}
```

Comment:
is this the same set up we have in Cloud?

Reply:
Yes, already removed the gsm one, but think we should leave the secrets stuff(if you want to follow the path of "each component is independent from the main chart)